model/admin_impl.go

```diff
@@ -16,12 +18,11 @@ func (repo *EntRepository) GetAdmins(ctx context.Context) ([]*Admin, error) {
 		return nil, err
 	}
 
-	admins := []*Admin{}
-	for _, u := range users {
-		admins = append(admins, &Admin{
+	admins := lo.Map(users, func(u *ent.User, insex int) *Admin {
```

Comment:
`index`使ってないので、`_`で受け取るようにしてください
他のところも:pray:

```suggestion
	admins := lo.Map(users, func(u *ent.User, _ int) *Admin {
```